ubuntudesign/maas-gui-vanilla-theme
# Update icon library

## Done

- Update icon library with latest VF icons
- Add titles to image to make identification easier
- Add static images to assets folder

## QA

- Pull branch, run npm install and gulp build
- Go to the demo - demo/index.html#icons to see the changes
- Make sure the icons listed in the card have been updated by comparing them to the icons at [http://ubuntudesign.github.io/maas-gui-vanilla-theme/#icons](http://ubuntudesign.github.io/maas-gui-vanilla-theme/#icons)

## Details

Fixes https://app.zenhub.com/workspace/o/canonicalltd/maas-design/issues/183
